base_layer/core/src/transactions/transaction_components/mod.rs

```diff
@@ -63,8 +63,6 @@ mod output_flags;
 mod rewind_result;
 mod side_chain_checkpoint_features;
 mod template_parameter;
-// TODO: in future, this module can be renamed
```

Comment:
Renames the parent module rather so this module can still be called Transaction as it contains the Transaction struct